newIDE/app/src/Export/Builds/BuildsList.js

```diff
@@ -41,27 +43,33 @@ const formatBuildText = (
   }
 };
 
-export default ({ builds, onDownload }: Props) => {
+export default ({ builds, userProfile, onDownload }: Props) => {
   return (
     <Column noMargin expand>
       <Line>
         <Column>
           <EmptyMessage>
             <Trans>
               This is the list of builds that you've done. Note that you can
-              download games generated during to 7 days, after which they are
+              download games generated during last 7 days, after which they are
```

Comment:
I think it should be **download games generated during the last 7 days, after which they are**. Because then it sounds grammatically correct

Reply:
Yep, "during the last 7 days" sounds better.